docs/content/concepts/transactions.mdx

```diff
@@ -2,20 +2,103 @@
 title: Transactions
 ---
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Scelerisque fermentum dui faucibus in ornare. Ut tristique et egestas quis ipsum suspendisse. Ac placerat vestibulum lectus mauris ultrices eros. Lorem ipsum dolor sit amet. Justo nec ultrices dui sapien eget mi proin sed libero. Nunc lobortis mattis aliquam faucibus. Nec ultrices dui sapien eget mi proin sed. Tellus elementum sagittis vitae et. Tellus at urna condimentum mattis pellentesque id nibh tortor. Sit amet venenatis urna cursus. Imperdiet sed euismod nisi porta lorem mollis aliquam ut. Malesuada fames ac turpis egestas maecenas pharetra convallis posuere.
+All updates to the Sui database happen via a transaction. This topic describes the transaction types supported by Sui and explains how their execution changes the ledger.
 
-## Vel orci
+## Programmable Transaction Blocks
```

Comment:
This is the only kind of transaction that's left -- "Move call", "Publish", "Transfer Objects", "Transfer Sui", etc are all encapsulated within this.  I would get rid of all of those sections below because they are/should be covered in the topic on PTBs.  Instead, this page should just contain something along these lines:

```
All updates to the Sui database happen via transactions. This topic describes the transaction types supported by Sui and explains how their execution changes the ledger.  There are only two kinds of transactions on Sui:

- Programmable transaction blocks, which can be submitted by anyone on the network.  For information on these transactions, see ...
- System transactions, which can only be submitted directly by validators and are responsible for keeping the network running (changing epochs, starting checkpoints, etc).

## Transaction metadata
...

## Transaction flow - example
...
```